third_party/com_github_grpc_ecosystem_grpc_gateway/WORKSPACE

```diff
@@ -0,0 +1 @@
+workspace(name = "com_github_grpc_ecosystem_grpc_gateway")
```

Comment:
You might be able to delete this file. I think Bazel creates it automatically in external repositories.